package.json

```diff
@@ -57,7 +57,7 @@
     "jslint": "prettier --check .",
     "jsonlint": "node scripts/lint/jsonlint.js",
     "svglint": "svglint icons/*.svg --ci",
-    "wslint": "editorconfig-checker -exclude \\.svg$",
+    "wslint": "editorconfig-checker -ignore-defaults",
```

Comment:
Is this change the reason a bunch of `.svg`s have their final newline removed? If so see #1682 for context on why they were excluded in the first place.

If we want to use editorconfig-checker here to check the type of newlines then I'm not sure what the best approach is. Forcing no final newlines has proven to be a real pain for some contributors with very little benefit. On the other hand, switching to requiring final newlines on SVGs would probably mean nearly 2000 file changes in a PR, causing a bunch of conflicts in the process... :thinking:

Reply:
Then we can use `.svglintrc.js` for linting SVG instead. Let's lint newline characters there.

BTW, the `-exclude  \\.svg$` won't take effect. Because `editorconfig-checker` already ignored them by default.

Reply:
> Then we can use .svglintrc.js for linting SVG instead. Let's lint newline characters there.

Please update the Pull Request accordingly, preferably including reverting the removal of final newlines from SVGs with an eye towards avoiding merge conflicts.

> BTW, the `-exclude \\.svg$` won't take effect. Because `editorconfig-checker` already ignored them by default.

I see thanks for pointing that out.

Reply:
Done.